conanfile.py

```diff
@@ -21,7 +21,7 @@ class SQLiteCppConan(ConanFile):
     default_options = {"shared": False,
                        "fPIC": True,
                        "lint": False,
-                       "sqlite3:threadsafe": 2,
+                       "sqlite3:threadsafe": 1,
```

Comment:
I don't think sqlite3 with threadsafe=1 is supported by SQLiteCpp. The SQLiteCpp documentation, https://github.com/SRombauts/SQLiteCpp/blob/master/README.md, says: "But SQLiteC++ does not support the fully thread-safe "Serialized" mode of SQLite, because of the way it shares the underlying SQLite precompiled statement in a custom shared pointer (See the inner class 'Statement::Ptr'). The serialized mode means  -DSQLITE_THREADSAFE=1 according to https://www.sqlite.org/threadsafe.html The multithread mode defined by  -DSQLITE_THREADSAFE=2 is supported. 
I have not tested the options myself, just followed the SQLiteCpp documentation. So I am not 100% sure about this.

Reply:
Hi! Thanks for your clarification! I'll request to make sqlite3:threadsafe=2 available in CCI.

Reply:
Good! Should we wait to see it fixed there or do you want to have the pull request approved and fix it later?